
Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-5wxfp[0m
{"level":"warn","ts":1591101542.7692523,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1591101542.7696676,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-55qxc[0m
{"level":"warn","ts":1591101543.8668761,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1591101543.868236,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-5gjck[0m
{"level":"warn","ts":1591101550.1308272,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1591101550.1325564,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1909-in-rdhw8[0m
{"level":"warn","ts":1591101550.411687,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1591101554.3798232,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:ff7274350e83f370d7b957f9dcbbc3174f482900,1909:538bf8dfbd1658be12ec589915e8dc9ad36b71db
DEBUG: ran git fetch --unshallow origin 538bf8dfbd1658be12ec589915e8dc9ad36b71db: ff7274350e83f370d7b957f9dcbbc3174f482900: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard ff7274350e83f370d7b957f9dcbbc3174f482900 in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge 538bf8dfbd1658be12ec589915e8dc9ad36b71db in 
Merged SHA 2124842017210238b97607878cb34e9bbdeb9c70 with commit message 'Merge commit '538bf8dfbd1658be12ec589915e8dc9ad36b71db'' into base branch master
Merged SHA 538bf8dfbd1658be12ec589915e8dc9ad36b71db with commit message 'Added extra steps to check if components are modified in tests' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1909
+ BUILD_NUMBER=5
+ PIPELINE_CONTEXT=integration
++ cat
+++ date
+ COMMENT_MSG='Tue Jun  2 12:39:21 UTC 2020
The logs for [integration] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1909/5.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1909 --build=5'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1909 '--comment=Tue Jun  2 12:39:21 UTC 2020
The logs for [integration] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1909/5.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1909 --build=5'

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1909-in-kjcm2[0m
{"level":"warn","ts":1591101566.299564,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1591101566.3030345,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1909-in-qh92f[0m
{"level":"warn","ts":1591101566.5550961,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1591101566.9213543,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mend-to-end[0m and container [32mstep-credential-initializer-twcgh[0m
{"level":"warn","ts":1591101596.6057243,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1591101596.6064038,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mend-to-end[0m and container [32mstep-working-dir-initializer-plkjs[0m
{"level":"warn","ts":1591101597.7592628,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1591101597.760835,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mend-to-end[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mend-to-end[0m and container [32mstep-create-dir-workspace-jfxbs[0m
{"level":"warn","ts":1591101606.054351,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1591101606.0565157,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mend-to-end[0m and container [32mstep-source-copy-workspace-chh7z[0m
{"level":"warn","ts":1591101606.3256238,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1591101616.7007885,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1909-in-s4l62-5[0m stage [32mend-to-end[0m and container [32mstep-test-end-to-end[0m
Test run type is: [base]
Starting Docker: docker.
[SETUP] Waiting for Docker to be ready, sleeping for 1 seconds ...
kind create cluster --config ../resources/kind_config.yaml
Creating cluster "kind" ...
 • Ensuring node image (kindest/node:v1.16.3) 🖼  ...
 ✓ Ensuring node image (kindest/node:v1.16.3) 🖼
 • Preparing nodes 📦  ...
 ✓ Preparing nodes 📦
 • Writing configuration 📜  ...
 ✓ Writing configuration 📜
 • Starting control-plane 🕹️  ...
 ✓ Starting control-plane 🕹️
 • Installing CNI 🔌  ...
 ✓ Installing CNI 🔌
 • Installing StorageClass 💾  ...
 ✓ Installing StorageClass 💾
 • Joining worker nodes 🚜  ...
 ✓ Joining worker nodes 🚜
Set kubectl context to "kind-kind"
You can now use your cluster with:

kubectl cluster-info --context kind-kind

Thanks for using kind! 😊
`kind get kubeconfig-path` is deprecated!

KIND will export and merge kubeconfig like kops, minikube, etc.
This command is now unnecessary and will be removed in a future release.

For more info see: https://github.com/kubernetes-sigs/kind/issues/1060
See also the output of `kind create cluster`

Files changed in python folder:
SKIPPING PYTHON IMAGE BUILD...
Files changed in operator folder:
SKIPPING OPERATOR IMAGE BUILD...
Files changed in engine folder:
SKIPPING ENGINE IMAGE BUILD...
Files changed in executor folder:
SKIPPING EXECUTOR IMAGE BUILD...
Build test models
make -C ../docker/fixed-model kind_load_images
make[1]: Entering directory '/workspace/source/testing/docker/fixed-model'
s2i build -E environment_rest_v1 . seldonio/seldon-core-s2i-python3:0.15 seldonio/fixed-model:0.1
---> Installing application source...
Build completed successfully
s2i build -E environment_rest_v2 . seldonio/seldon-core-s2i-python3:0.15 seldonio/fixed-model:0.2
---> Installing application source...
Build completed successfully
kind load -v 3 docker-image seldonio/fixed-model:0.1
Image: "seldonio/fixed-model:0.1" with ID "sha256:096aad4aa5c91070edcba881ad40ba55a838fc854ce4e894e1c39b7e7f2692ab" not present on node "kind-worker"
Image: "seldonio/fixed-model:0.1" with ID "sha256:096aad4aa5c91070edcba881ad40ba55a838fc854ce4e894e1c39b7e7f2692ab" not present on node "kind-control-plane"
kind load -v 3 docker-image seldonio/fixed-model:0.2
Image: "seldonio/fixed-model:0.2" with ID "sha256:d304c980b7475eadf43890d544073b427763f19b4aaaa94b5c3bcded33ff0f01" not present on node "kind-worker"
Image: "seldonio/fixed-model:0.2" with ID "sha256:d304c980b7475eadf43890d544073b427763f19b4aaaa94b5c3bcded33ff0f01" not present on node "kind-control-plane"
make[1]: Leaving directory '/workspace/source/testing/docker/fixed-model'
make -C ../docker/echo-model kind_load_image
make[1]: Entering directory '/workspace/source/testing/docker/echo-model'
s2i build \
	-E environment_rest \
	.\
	seldonio/seldon-core-s2i-python3:1.1.1-SNAPSHOT\
	seldonio/echo-model:0.1
---> Installing application source...
Build completed successfully
kind load -v 3 docker-image seldonio/echo-model:0.1
Image: "seldonio/echo-model:0.1" with ID "sha256:4bddcf88257ebc1f8f7417852502a071dd9af3e20b3dd2ff1a63b988981e0c2d" not present on node "kind-worker"
Image: "seldonio/echo-model:0.1" with ID "sha256:4bddcf88257ebc1f8f7417852502a071dd9af3e20b3dd2ff1a63b988981e0c2d" not present on node "kind-control-plane"
make[1]: Leaving directory '/workspace/source/testing/docker/echo-model'
Files changed in prepackaged folder:
SKIPPING PREPACKAGED IMAGE BUILD...
Files changed in alibi folder:
SKIPPING ALIBI IMAGE BUILD...
Files changed in misc folders:
SKIPPING MISC IMAGE BUILD...
kubectl create namespace seldon || echo "Namespace seldon already exists"
namespace/seldon created
kubectl create namespace test1 || echo "Namespace test1 already exists"
namespace/test1 created
helm repo add stable https://kubernetes-charts.storage.googleapis.com/
"stable" has been added to your repositories
helm repo add seldonio https://storage.googleapis.com/seldon-charts
"seldonio" has been added to your repositories
helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
"jaegertracing" has been added to your repositories
helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "jaegertracing" chart repository
...Successfully got an update from the "seldonio" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈ 
helm install ambassador \
	stable/ambassador \
	-f ../resources/ambassador_values.yaml \
	--set crds.keep=false \
	--namespace seldon \
	--set replicaCount=1 \
	--wait
WARNING: This chart is deprecated
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
NAME: ambassador
LAST DEPLOYED: Tue Jun  2 12:48:05 2020
NAMESPACE: seldon
STATUS: deployed
REVISION: 1
NOTES:
DEPRECATED!! This version of the chart has been deprecated.
Please use https://github.com/datawire/ambassador-chart

Congratulations! You've successfully installed Ambassador.

For help, visit our Slack at https://d6e.co/slack or view the documentation online at https://www.getambassador.io.

To get the IP address of Ambassador, run the following commands:
  export NODE_PORT=$(kubectl get --namespace seldon -o jsonpath="{.spec.ports[0].nodePort}" services ambassador)
  export NODE_IP=$(kubectl get nodes --namespace seldon -o jsonpath="{.items[0].status.addresses[0].address}")
  echo http://$NODE_IP:$NODE_PORT
kubectl apply -f istio-1.4.2.yaml
clusterrole.rbac.authorization.k8s.io/istio-reader-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-reader-istio-system created
customresourcedefinition.apiextensions.k8s.io/attributemanifests.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/clusterrbacconfigs.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/destinationrules.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/envoyfilters.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/gateways.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/httpapispecbindings.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/httpapispecs.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/meshpolicies.authentication.istio.io created
customresourcedefinition.apiextensions.k8s.io/policies.authentication.istio.io created
customresourcedefinition.apiextensions.k8s.io/quotaspecbindings.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/quotaspecs.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/rbacconfigs.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/rules.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/serviceentries.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/servicerolebindings.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/serviceroles.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/virtualservices.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/adapters.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/instances.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/templates.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/handlers.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/sidecars.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/authorizationpolicies.security.istio.io created
namespace/istio-system created
serviceaccount/istio-reader-service-account created
clusterrole.rbac.authorization.k8s.io/istio-citadel-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-citadel-istio-system created
deployment.apps/istio-citadel created
poddisruptionbudget.policy/istio-citadel created
service/istio-citadel created
serviceaccount/istio-citadel-service-account created
deployment.apps/istio-egressgateway created
poddisruptionbudget.policy/istio-egressgateway created
gateway.networking.istio.io/istio-multicluster-egressgateway created
virtualservice.networking.istio.io/istio-multicluster-egressgateway created
envoyfilter.networking.istio.io/istio-multicluster-egressgateway created
destinationrule.networking.istio.io/istio-multicluster-destinationrule created
service/istio-egressgateway created
role.rbac.authorization.k8s.io/istio-egressgateway-sds created
rolebinding.rbac.authorization.k8s.io/istio-egressgateway-sds created
serviceaccount/istio-egressgateway-service-account created
clusterrole.rbac.authorization.k8s.io/istio-galley-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-galley-*****-role-binding-istio-system created
configmap/istio-mesh-galley created
configmap/istio-galley-configuration created
deployment.apps/istio-galley created
poddisruptionbudget.policy/istio-galley created
service/istio-galley created
serviceaccount/istio-galley-service-account created
configmap/istio-grafana-configuration-dashboards-citadel-dashboard created
configmap/istio-grafana-configuration-dashboards-galley-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-mesh-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-performance-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-service-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-workload-dashboard created
configmap/istio-grafana-configuration-dashboards-mixer-dashboard created
configmap/istio-grafana-configuration-dashboards-pilot-dashboard created
configmap/istio-grafana created
deployment.apps/grafana created
policy.authentication.istio.io/grafana-ports-mtls-disabled created
service/grafana created
deployment.apps/istio-ingressgateway created
gateway.networking.istio.io/ingressgateway created
poddisruptionbudget.policy/ingressgateway created
service/istio-ingressgateway created
serviceaccount/istio-ingressgateway-service-account created
sidecar.networking.istio.io/default created
clusterrole.rbac.authorization.k8s.io/istio-sidecar-injector-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-sidecar-injector-*****-role-binding-istio-system created
configmap/injector-mesh created
deployment.apps/istio-sidecar-injector created
mutatingwebhookconfiguration.admissionregistration.k8s.io/istio-sidecar-injector created
poddisruptionbudget.policy/istio-sidecar-injector created
service/istio-sidecar-injector created
serviceaccount/istio-sidecar-injector-service-account created
configmap/istio-sidecar-injector created
clusterrole.rbac.authorization.k8s.io/kiali created
clusterrole.rbac.authorization.k8s.io/kiali-viewer created
clusterrolebinding.rbac.authorization.k8s.io/kiali created
configmap/kiali created
secret/kiali created
deployment.apps/kiali created
service/kiali created
serviceaccount/kiali-service-account created
clusterrole.rbac.authorization.k8s.io/istio-pilot-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-pilot-istio-system created
configmap/pilot-envoy-config created
configmap/istio created
deployment.apps/istio-pilot created
meshpolicy.authentication.istio.io/default created
poddisruptionbudget.policy/istio-pilot created
service/istio-pilot created
serviceaccount/istio-pilot-service-account created
clusterrole.rbac.authorization.k8s.io/istio-policy created
clusterrolebinding.rbac.authorization.k8s.io/istio-policy-*****-role-binding-istio-system created
destinationrule.networking.istio.io/istio-policy created
deployment.apps/istio-policy created
poddisruptionbudget.policy/istio-policy created
service/istio-policy created
serviceaccount/istio-policy-service-account created
clusterrole.rbac.authorization.k8s.io/prometheus-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-istio-system created
configmap/prometheus created
deployment.apps/prometheus created
service/prometheus created
serviceaccount/prometheus created
horizontalpodautoscaler.autoscaling/istio-telemetry created
clusterrole.rbac.authorization.k8s.io/istio-mixer-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-mixer-*****-role-binding-istio-system created
attributemanifest.config.istio.io/istioproxy created
attributemanifest.config.istio.io/kubernetes created
handler.config.istio.io/stdio created
instance.config.istio.io/accesslog created
instance.config.istio.io/tcpaccesslog created
rule.config.istio.io/stdio created
rule.config.istio.io/stdiotcp created
instance.config.istio.io/requestcount created
instance.config.istio.io/requestduration created
instance.config.istio.io/requestsize created
instance.config.istio.io/responsesize created
instance.config.istio.io/tcpbytesent created
instance.config.istio.io/tcpbytereceived created
instance.config.istio.io/tcpconnectionsopened created
instance.config.istio.io/tcpconnectionsclosed created
handler.config.istio.io/prometheus created
rule.config.istio.io/promhttp created
rule.config.istio.io/promtcp created
rule.config.istio.io/promtcpconnectionopen created
rule.config.istio.io/promtcpconnectionclosed created
handler.config.istio.io/kubernetesenv created
rule.config.istio.io/kubeattrgenrulerule created
rule.config.istio.io/tcpkubeattrgenrulerule created
instance.config.istio.io/attributes created
destinationrule.networking.istio.io/istio-telemetry created
configmap/telemetry-envoy-config created
deployment.apps/istio-telemetry created
poddisruptionbudget.policy/istio-telemetry created
service/istio-telemetry created
serviceaccount/istio-mixer-service-account created
deployment.apps/istio-tracing created
service/jaeger-query created
service/jaeger-collector created
service/jaeger-agent created
service/zipkin created
service/tracing created
kubectl rollout status deployment.apps/istio-ingressgateway -n istio-system
Waiting for deployment "istio-ingressgateway" rollout to finish: 0 of 1 updated replicas are available...
deployment "istio-ingressgateway" successfully rolled out
kubectl rollout status deployment.apps/istio-pilot -n istio-system
deployment "istio-pilot" successfully rolled out
kubectl rollout status deployment.apps/istio-citadel -n istio-system
deployment "istio-citadel" successfully rolled out
kubectl create -f ../resources/seldon-gateway.yaml -n istio-system
gateway.networking.istio.io/seldon-gateway created
helm install jaeger-operator \
	jaegertracing/jaeger-operator \
	--set rbac.clusterRole=true \
	--namespace seldon \
	--wait
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
NAME: jaeger-operator
LAST DEPLOYED: Tue Jun  2 12:50:50 2020
NAMESPACE: seldon
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
jaeger-operator is installed.


Check the jaeger-operator logs
  export POD=$(kubectl get pods-l app.kubernetes.io/instance=jaeger-operator -lapp.kubernetes.io/name=jaeger-operator --namespace seldon --output name)
  kubectl logs $POD --namespace=seldon
kubectl apply -f ../resources/jaeger.yaml --namespace seldon
jaeger.jaegertracing.io/jaeger created
mapping.getambassador.io/jaeger created
cd ../../operator && make install-cert-manager
make[1]: Entering directory '/workspace/source/operator'
kubectl create namespace cert-manager || echo "Namespace cert-manager-exists"
namespace/cert-manager created
kubectl label namespace cert-manager cert-manager.io/disable-validation=true || echo "namespace cert-manager-already labelled"
namespace/cert-manager labeled
kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.12.0/cert-manager.yaml
customresourcedefinition.apiextensions.k8s.io/certificaterequests.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/certificates.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/challenges.acme.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/clusterissuers.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/issuers.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/orders.acme.cert-manager.io created
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
namespace/cert-manager configured
serviceaccount/cert-manager-cainjector created
serviceaccount/cert-manager created
serviceaccount/cert-manager-webhook created
clusterrole.rbac.authorization.k8s.io/cert-manager-cainjector created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-cainjector created
role.rbac.authorization.k8s.io/cert-manager-cainjector:leaderelection created
rolebinding.rbac.authorization.k8s.io/cert-manager-cainjector:leaderelection created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-webhook:auth-delegator created
rolebinding.rbac.authorization.k8s.io/cert-manager-webhook:webhook-authentication-reader created
clusterrole.rbac.authorization.k8s.io/cert-manager-webhook:webhook-requester created
role.rbac.authorization.k8s.io/cert-manager:leaderelection created
rolebinding.rbac.authorization.k8s.io/cert-manager:leaderelection created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-issuers created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-clusterissuers created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-certificates created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-orders created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-challenges created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-ingress-shim created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-issuers created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-clusterissuers created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-certificates created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-orders created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-challenges created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-ingress-shim created
clusterrole.rbac.authorization.k8s.io/cert-manager-view created
clusterrole.rbac.authorization.k8s.io/cert-manager-edit created
service/cert-manager created
service/cert-manager-webhook created
deployment.apps/cert-manager-cainjector created
deployment.apps/cert-manager created
deployment.apps/cert-manager-webhook created
mutatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook created
validatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook created
kubectl rollout status deployment.apps/cert-manager -n cert-manager
Waiting for deployment "cert-manager" rollout to finish: 0 of 1 updated replicas are available...
deployment "cert-manager" successfully rolled out
kubectl rollout status deployment.apps/cert-manager-cainjector -n cert-manager
deployment "cert-manager-cainjector" successfully rolled out
kubectl rollout status deployment.apps/cert-manager-webhook -n cert-manager
Waiting for deployment "cert-manager-webhook" rollout to finish: 0 of 1 updated replicas are available...
deployment "cert-manager-webhook" successfully rolled out
make[1]: Leaving directory '/workspace/source/operator'
sleep 5 #https://github.com/jetstack/cert-manager/issues/2273
kubectl create namespace seldon-system || echo "namespace seldon-system exists"
namespace/seldon-system created
helm install seldon \
	../../helm-charts/seldon-core-operator \
	--namespace seldon-system \
	--set istio.enabled=true \
	--set istio.gateway=istio-system/seldon-gateway \
	--set certManager.enabled=false \
	--set executor.enabled=true \
	--wait
NAME: seldon
LAST DEPLOYED: Tue Jun  2 12:51:58 2020
NAMESPACE: seldon-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
kubectl config set-context $(kubectl config current-context) --namespace=seldon
Context "kind-kind" modified.
kubectl apply -f metrics.yaml
clusterrole.rbac.authorization.k8s.io/system:aggregated-metrics-reader created
clusterrolebinding.rbac.authorization.k8s.io/metrics-server:system:auth-delegator created
rolebinding.rbac.authorization.k8s.io/metrics-server-auth-reader created
apiservice.apiregistration.k8s.io/v1beta1.metrics.k8s.io created
serviceaccount/metrics-server created
deployment.apps/metrics-server created
service/metrics-server created
clusterrole.rbac.authorization.k8s.io/system:metrics-server created
clusterrolebinding.rbac.authorization.k8s.io/system:metrics-server created
make: Entering directory '/workspace/source/python'
pip install -e . -r requirements-dev.txt
Obtaining file:///workspace/source/python
Collecting Flask<2.0.0
  Downloading Flask-1.1.2-py2.py3-none-any.whl (94 kB)
Collecting Flask-cors<4.0.0
  Downloading Flask_Cors-3.0.8-py2.py3-none-any.whl (14 kB)
Collecting redis<4.0.0
  Downloading redis-3.5.3-py2.py3-none-any.whl (72 kB)
Requirement already satisfied: requests<3.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 4)) (2.23.0)
Requirement already satisfied: numpy<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 5)) (1.18.4)
Collecting flatbuffers<2.0.0
  Downloading flatbuffers-1.12-py2.py3-none-any.whl (15 kB)
Requirement already satisfied: protobuf<4.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 7)) (3.11.3)
Requirement already satisfied: grpcio<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 8)) (1.28.1)
Collecting Flask-OpenTracing<1.2.0,>=1.1.0
  Downloading Flask-OpenTracing-1.1.0.tar.gz (8.2 kB)
Collecting opentracing<2.3.0,>=2.2.0
  Downloading opentracing-2.2.0.tar.gz (47 kB)
Collecting jaeger-client<4.2.0,>=4.1.0
  Downloading jaeger-client-4.1.0.tar.gz (80 kB)
Collecting grpcio-opentracing<1.2.0,>=1.1.4
  Downloading grpcio_opentracing-1.1.4-py3-none-any.whl (14 kB)
Collecting pyaml<20.0.0
  Downloading pyaml-19.12.0-py2.py3-none-any.whl (17 kB)
Collecting gunicorn<20.1.0,>=19.9.0
  Downloading gunicorn-20.0.4-py2.py3-none-any.whl (77 kB)
Collecting minio<6.0.0,>=4.0.9
  Downloading minio-5.0.10-py2.py3-none-any.whl (75 kB)
Collecting azure-storage-blob<3.0.0,>=2.0.1
  Downloading azure_storage_blob-2.1.0-py2.py3-none-any.whl (88 kB)
Requirement already satisfied: setuptools>=41.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 17)) (46.3.0)
Collecting google-cloud-storage>=1.16.0
  Downloading google_cloud_storage-1.28.1-py2.py3-none-any.whl (80 kB)
Requirement already satisfied: prometheus_client<0.8.0,>=0.7.1 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 19)) (0.7.1)
Collecting black==19.10b0
  Downloading black-19.10b0-py36-none-any.whl (97 kB)
Collecting flake8==3.8.1
  Downloading flake8-3.8.1-py2.py3-none-any.whl (72 kB)
Collecting mypy<0.771
  Downloading mypy-0.770-cp36-cp36m-manylinux1_x86_64.whl (21.7 MB)
Collecting Pillow==7.1.1
  Downloading Pillow-7.1.1-cp36-cp36m-manylinux1_x86_64.whl (2.1 MB)
Collecting pytest==5.4.2
  Downloading pytest-5.4.2-py3-none-any.whl (247 kB)
Collecting pytest-cov==2.9.0
  Downloading pytest_cov-2.9.0-py2.py3-none-any.whl (19 kB)
Collecting tox<4.0.0
  Downloading tox-3.15.1-py2.py3-none-any.whl (82 kB)
Collecting grpcio-tools==1.27.2
  Downloading grpcio_tools-1.27.2-cp36-cp36m-manylinux2010_x86_64.whl (2.4 MB)
Collecting mypy-protobuf==1.20
  Downloading mypy_protobuf-1.20-py3-none-any.whl (7.4 kB)
Collecting tenacity==6.2.0
  Downloading tenacity-6.2.0-py2.py3-none-any.whl (24 kB)
Collecting coverage==5.1
  Downloading coverage-5.1-cp36-cp36m-manylinux1_x86_64.whl (227 kB)
Collecting pandas==1.0.1
  Downloading pandas-1.0.1-cp36-cp36m-manylinux1_x86_64.whl (10.1 MB)
Collecting pip-licenses==2.2.0
  Downloading pip_licenses-2.2.0-py3-none-any.whl (14 kB)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->-r requirements.txt (line 1)) (2.11.2)
Requirement already satisfied: Werkzeug>=0.15 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->-r requirements.txt (line 1)) (1.0.1)
Collecting itsdangerous>=0.24
  Downloading itsdangerous-1.1.0-py2.py3-none-any.whl (16 kB)
Collecting click>=5.1
  Downloading click-7.1.2-py2.py3-none-any.whl (82 kB)
Requirement already satisfied: Six in /usr/local/lib/python3.6/site-packages (from Flask-cors<4.0.0->-r requirements.txt (line 2)) (1.14.0)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 4)) (1.25.9)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 4)) (2020.4.5.1)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 4)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 4)) (2.9)
Collecting threadloop<2,>=1
  Downloading threadloop-1.0.2.tar.gz (4.9 kB)
Collecting thrift
  Downloading thrift-0.13.0.tar.gz (59 kB)
Collecting tornado<6,>=4.3
  Downloading tornado-5.1.1.tar.gz (516 kB)
Requirement already satisfied: PyYAML in /usr/local/lib/python3.6/site-packages (from pyaml<20.0.0->-r requirements.txt (line 13)) (5.3.1)
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->-r requirements.txt (line 15)) (2.8.1)
Requirement already satisfied: pytz in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->-r requirements.txt (line 15)) (2020.1)
Collecting configparser
  Downloading configparser-5.0.0-py3-none-any.whl (22 kB)
Collecting azure-common>=1.1.5
  Downloading azure_common-1.1.25-py2.py3-none-any.whl (12 kB)
Collecting azure-storage-common~=2.1
  Downloading azure_storage_common-2.1.0-py2.py3-none-any.whl (47 kB)
Collecting google-cloud-core<2.0dev,>=1.2.0
  Downloading google_cloud_core-1.3.0-py2.py3-none-any.whl (26 kB)
Collecting google-resumable-media<0.6dev,>=0.5.0
  Downloading google_resumable_media-0.5.0-py2.py3-none-any.whl (38 kB)
Collecting google-auth<2.0dev,>=1.11.0
  Downloading google_auth-1.16.0-py2.py3-none-any.whl (90 kB)
Collecting appdirs
  Downloading appdirs-1.4.4-py2.py3-none-any.whl (9.6 kB)
Collecting pathspec<1,>=0.6
  Downloading pathspec-0.8.0-py2.py3-none-any.whl (28 kB)
Collecting toml>=0.9.4
  Downloading toml-0.10.1-py2.py3-none-any.whl (19 kB)
Collecting typed-ast>=1.4.0
  Downloading typed_ast-1.4.1-cp36-cp36m-manylinux1_x86_64.whl (737 kB)
Collecting regex
  Downloading regex-2020.5.14-cp36-cp36m-manylinux2010_x86_64.whl (675 kB)
Requirement already satisfied: attrs>=18.1.0 in /usr/local/lib/python3.6/site-packages (from black==19.10b0->-r requirements-dev.txt (line 2)) (19.3.0)
Collecting mccabe<0.7.0,>=0.6.0
  Downloading mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
Collecting pyflakes<2.3.0,>=2.2.0
  Downloading pyflakes-2.2.0-py2.py3-none-any.whl (66 kB)
Collecting pycodestyle<2.7.0,>=2.6.0a1
  Downloading pycodestyle-2.6.0-py2.py3-none-any.whl (41 kB)
Requirement already satisfied: importlib-metadata; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from flake8==3.8.1->-r requirements-dev.txt (line 3)) (1.6.0)
Collecting typing-extensions>=3.7.4
  Downloading typing_extensions-3.7.4.2-py3-none-any.whl (22 kB)
Collecting mypy-extensions<0.5.0,>=0.4.3
  Downloading mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
Requirement already satisfied: pluggy<1.0,>=0.12 in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (0.13.1)
Requirement already satisfied: py>=1.5.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (1.8.1)
Requirement already satisfied: packaging in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (20.3)
Requirement already satisfied: wcwidth in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (0.1.9)
Requirement already satisfied: more-itertools>=4.0.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (8.2.0)
Collecting virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0
  Downloading virtualenv-20.0.21-py2.py3-none-any.whl (4.7 MB)
Collecting filelock>=3.0.0
  Downloading filelock-3.0.12-py3-none-any.whl (7.6 kB)
Collecting PTable
  Downloading PTable-0.9.2.tar.gz (31 kB)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/site-packages (from Jinja2>=2.10.1->Flask<2.0.0->-r requirements.txt (line 1)) (1.1.1)
Collecting cryptography
  Downloading cryptography-2.9.2-cp35-abi3-manylinux2010_x86_64.whl (2.7 MB)
Collecting google-api-core<2.0.0dev,>=1.16.0
  Downloading google_api_core-1.17.0-py2.py3-none-any.whl (70 kB)
Collecting pyasn1-modules>=0.2.1
  Downloading pyasn1_modules-0.2.8-py2.py3-none-any.whl (155 kB)
Collecting rsa<4.1,>=3.1.4
  Downloading rsa-4.0-py2.py3-none-any.whl (38 kB)
Collecting cachetools<5.0,>=2.0.0
  Downloading cachetools-4.1.0-py3-none-any.whl (10 kB)
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/site-packages (from importlib-metadata; python_version < "3.8"->flake8==3.8.1->-r requirements-dev.txt (line 3)) (3.1.0)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.6/site-packages (from packaging->pytest==5.4.2->-r requirements-dev.txt (line 6)) (2.4.7)
Collecting importlib-resources<2,>=1.0; python_version < "3.7"
  Downloading importlib_resources-1.5.0-py2.py3-none-any.whl (21 kB)
Collecting distlib<1,>=0.3.0
  Downloading distlib-0.3.0.zip (571 kB)
Collecting cffi!=1.11.3,>=1.8
  Downloading cffi-1.14.0-cp36-cp36m-manylinux1_x86_64.whl (399 kB)
Collecting googleapis-common-protos<2.0dev,>=1.6.0
  Downloading googleapis-common-protos-1.51.0.tar.gz (35 kB)
Collecting pyasn1<0.5.0,>=0.4.6
  Downloading pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
Collecting pycparser
  Downloading pycparser-2.20-py2.py3-none-any.whl (112 kB)
Building wheels for collected packages: Flask-OpenTracing, opentracing, jaeger-client, threadloop, thrift, tornado, PTable, distlib, googleapis-common-protos
  Building wheel for Flask-OpenTracing (setup.py): started
  Building wheel for Flask-OpenTracing (setup.py): finished with status 'done'
  Created wheel for Flask-OpenTracing: filename=Flask_OpenTracing-1.1.0-py3-none-any.whl size=9070 sha256=45280a684f830b7aa2962ed42479d117dbc8dab5e208d3d9532f7de94efe59cc
  Stored in directory: /builder/home/.cache/pip/wheels/ad/4b/2d/24ff0da0a0b53c7c77ce59b843bcceaf644c88703241e59615
  Building wheel for opentracing (setup.py): started
  Building wheel for opentracing (setup.py): finished with status 'done'
  Created wheel for opentracing: filename=opentracing-2.2.0-py3-none-any.whl size=49319 sha256=504eacdf811860cd15e382efc6bb8d84f2122852f6396cb79219f6589a06eaac
  Stored in directory: /builder/home/.cache/pip/wheels/39/40/44/8bace79f4514e99786236c31f1df8d1b814ff02c1e08b1d697
  Building wheel for jaeger-client (setup.py): started
  Building wheel for jaeger-client (setup.py): finished with status 'done'
  Created wheel for jaeger-client: filename=jaeger_client-4.1.0-py3-none-any.whl size=64309 sha256=60d5a7e30a2613296967c0924b3910627e61fffb274351046eb97e689855efaa
  Stored in directory: /builder/home/.cache/pip/wheels/e9/9b/8c/503d0cc13b39a551c054515683ba1d15b40324c863dc442e66
  Building wheel for threadloop (setup.py): started
  Building wheel for threadloop (setup.py): finished with status 'done'
  Created wheel for threadloop: filename=threadloop-1.0.2-py3-none-any.whl size=3423 sha256=765c6b143080c111ffff9cc92ebad56d499e8e1c482d5f4602761f4ef5370e5a
  Stored in directory: /builder/home/.cache/pip/wheels/02/54/65/9f87de48fe8fcaaee30f279973d946ad55f9df56b93b3e78da
  Building wheel for thrift (setup.py): started
  Building wheel for thrift (setup.py): finished with status 'done'
  Created wheel for thrift: filename=thrift-0.13.0-cp36-cp36m-linux_x86_64.whl size=483924 sha256=18427258f67005764d9cde22e4203d62562c665bb1741c10664dc32f420976c5
  Stored in directory: /builder/home/.cache/pip/wheels/e0/38/fc/472fe18756b177b42096961f8bd3ff2dc5c5620ac399fce52d
  Building wheel for tornado (setup.py): started
  Building wheel for tornado (setup.py): finished with status 'done'
  Created wheel for tornado: filename=tornado-5.1.1-cp36-cp36m-linux_x86_64.whl size=462645 sha256=3d78c1f4316419386bae2773553a29656314da3274d9895372ad5e51aeb8c1c5
  Stored in directory: /builder/home/.cache/pip/wheels/64/74/71/e7a0d0eb4fc42cb20a17aec36f8f0b5b8c5e1ec19c701fd34a
  Building wheel for PTable (setup.py): started
  Building wheel for PTable (setup.py): finished with status 'done'
  Created wheel for PTable: filename=PTable-0.9.2-py3-none-any.whl size=22907 sha256=c3b2d2cd670589d9fc869df4ba32674836ba444c5d07823d715f47b95d36e802
  Stored in directory: /builder/home/.cache/pip/wheels/f3/65/67/71f473ec87ea4554d1bdfb1b5128cfe1414c2b113b72a1ee8e
  Building wheel for distlib (setup.py): started
  Building wheel for distlib (setup.py): finished with status 'done'
  Created wheel for distlib: filename=distlib-0.3.0-py3-none-any.whl size=340427 sha256=ca07d9e81c9b02a75f779776e2b9df55294eeb3b7aa5ef3c9848b65b98d14bea
  Stored in directory: /builder/home/.cache/pip/wheels/33/d9/71/e4e3cac73529e1947df418af0f140cd7589d5d9ec0e17ecfc2
  Building wheel for googleapis-common-protos (setup.py): started
  Building wheel for googleapis-common-protos (setup.py): finished with status 'done'
  Created wheel for googleapis-common-protos: filename=googleapis_common_protos-1.51.0-py3-none-any.whl size=77592 sha256=8468cc7904ea16da35b0cb6493423bdace471a676485287cd571608592c27ba5
  Stored in directory: /builder/home/.cache/pip/wheels/35/8d/af/a922cb18800b31fadac3523cadf6c1efdf233b788fe7a4da70
Successfully built Flask-OpenTracing opentracing jaeger-client threadloop thrift tornado PTable distlib googleapis-common-protos
ERROR: scikit-image 0.17.2 has requirement pillow!=7.1.0,!=7.1.1,>=4.3.0, but you'll have pillow 7.1.1 which is incompatible.
Installing collected packages: itsdangerous, click, Flask, Flask-cors, redis, flatbuffers, opentracing, Flask-OpenTracing, tornado, threadloop, thrift, jaeger-client, grpcio-opentracing, pyaml, gunicorn, configparser, minio, azure-common, pycparser, cffi, cryptography, azure-storage-common, azure-storage-blob, googleapis-common-protos, pyasn1, pyasn1-modules, rsa, cachetools, google-auth, google-api-core, google-cloud-core, google-resumable-media, google-cloud-storage, appdirs, pathspec, toml, typed-ast, regex, black, mccabe, pyflakes, pycodestyle, flake8, typing-extensions, mypy-extensions, mypy, Pillow, pytest, coverage, pytest-cov, filelock, importlib-resources, distlib, virtualenv, tox, grpcio-tools, mypy-protobuf, tenacity, pandas, PTable, pip-licenses, seldon-core
  Attempting uninstall: tornado
    Found existing installation: tornado 6.0.4
    Uninstalling tornado-6.0.4:
      Successfully uninstalled tornado-6.0.4
  Attempting uninstall: Pillow
    Found existing installation: Pillow 7.1.2
    Uninstalling Pillow-7.1.2:
      Successfully uninstalled Pillow-7.1.2
  Attempting uninstall: pytest
    Found existing installation: pytest 5.3.1
    Uninstalling pytest-5.3.1:
      Successfully uninstalled pytest-5.3.1
  Attempting uninstall: coverage
    Found existing installation: coverage 4.5.4
    Uninstalling coverage-4.5.4:
      Successfully uninstalled coverage-4.5.4
  Attempting uninstall: pytest-cov
    Found existing installation: pytest-cov 2.8.1
    Uninstalling pytest-cov-2.8.1:
      Successfully uninstalled pytest-cov-2.8.1
  Attempting uninstall: grpcio-tools
    Found existing installation: grpcio-tools 1.28.1
    Uninstalling grpcio-tools-1.28.1:
      Successfully uninstalled grpcio-tools-1.28.1
  Attempting uninstall: tenacity
    Found existing installation: tenacity 6.0.0
    Uninstalling tenacity-6.0.0:
      Successfully uninstalled tenacity-6.0.0
  Attempting uninstall: pandas
    Found existing installation: pandas 1.0.3
    Uninstalling pandas-1.0.3:
      Successfully uninstalled pandas-1.0.3
  Running setup.py develop for seldon-core
Successfully installed Flask-1.1.2 Flask-OpenTracing-1.1.0 Flask-cors-3.0.8 PTable-0.9.2 Pillow-7.1.1 appdirs-1.4.4 azure-common-1.1.25 azure-storage-blob-2.1.0 azure-storage-common-2.1.0 black-19.10b0 cachetools-4.1.0 cffi-1.14.0 click-7.1.2 configparser-5.0.0 coverage-5.1 cryptography-2.9.2 distlib-0.3.0 filelock-3.0.12 flake8-3.8.1 flatbuffers-1.12 google-api-core-1.17.0 google-auth-1.16.0 google-cloud-core-1.3.0 google-cloud-storage-1.28.1 google-resumable-media-0.5.0 googleapis-common-protos-1.51.0 grpcio-opentracing-1.1.4 grpcio-tools-1.27.2 gunicorn-20.0.4 importlib-resources-1.5.0 itsdangerous-1.1.0 jaeger-client-4.1.0 mccabe-0.6.1 minio-5.0.10 mypy-0.770 mypy-extensions-0.4.3 mypy-protobuf-1.20 opentracing-2.2.0 pandas-1.0.1 pathspec-0.8.0 pip-licenses-2.2.0 pyaml-19.12.0 pyasn1-0.4.8 pyasn1-modules-0.2.8 pycodestyle-2.6.0 pycparser-2.20 pyflakes-2.2.0 pytest-5.4.2 pytest-cov-2.9.0 redis-3.5.3 regex-2020.5.14 rsa-4.0 seldon-core tenacity-6.2.0 threadloop-1.0.2 thrift-0.13.0 toml-0.10.1 tornado-5.1.1 tox-3.15.1 typed-ast-1.4.1 typing-extensions-3.7.4.2 virtualenv-20.0.21
WARNING: You are using pip version 20.1; however, version 20.1.1 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
make: Leaving directory '/workspace/source/python'
cp ../../proto/prediction.proto ./proto
cd ../../proto/tensorflow && make create_protos
make[1]: Entering directory '/workspace/source/proto/tensorflow'
./create-protos.sh
Downloading proto files for master
make[1]: Leaving directory '/workspace/source/proto/tensorflow'
cp -vr ../../proto/tensorflow/tensorflow .
'../../proto/tensorflow/tensorflow' -> './tensorflow'
'../../proto/tensorflow/tensorflow/core' -> './tensorflow/core'
'../../proto/tensorflow/tensorflow/core/framework' -> './tensorflow/core/framework'
'../../proto/tensorflow/tensorflow/core/framework/types.proto' -> './tensorflow/core/framework/types.proto'
'../../proto/tensorflow/tensorflow/core/framework/resource_handle.proto' -> './tensorflow/core/framework/resource_handle.proto'
'../../proto/tensorflow/tensorflow/core/framework/tensor_shape.proto' -> './tensorflow/core/framework/tensor_shape.proto'
'../../proto/tensorflow/tensorflow/core/framework/tensor.proto' -> './tensorflow/core/framework/tensor.proto'
python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. ./proto/prediction.proto
cd ../../notebooks && make build_protos
make[1]: Entering directory '/workspace/source/notebooks'
cd ../proto/tensorflow && make create_protos
make[2]: Entering directory '/workspace/source/proto/tensorflow'
make[2]: Nothing to be done for 'create_protos'.
make[2]: Leaving directory '/workspace/source/proto/tensorflow'
cp -vr ../proto/tensorflow/tensorflow .
'../proto/tensorflow/tensorflow' -> './tensorflow'
'../proto/tensorflow/tensorflow/core' -> './tensorflow/core'
'../proto/tensorflow/tensorflow/core/framework' -> './tensorflow/core/framework'
'../proto/tensorflow/tensorflow/core/framework/types.proto' -> './tensorflow/core/framework/types.proto'
'../proto/tensorflow/tensorflow/core/framework/resource_handle.proto' -> './tensorflow/core/framework/resource_handle.proto'
'../proto/tensorflow/tensorflow/core/framework/tensor_shape.proto' -> './tensorflow/core/framework/tensor_shape.proto'
'../proto/tensorflow/tensorflow/core/framework/tensor.proto' -> './tensorflow/core/framework/tensor.proto'
cp ../proto/prediction.proto ./proto
python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. ./proto/prediction.proto
make[1]: Leaving directory '/workspace/source/notebooks'
cd ../../proto/k8s && make create_protos
make[1]: Entering directory '/workspace/source/proto/k8s'
./create-k8s-protos.sh
Downloading proto files for master
Munging proto file packages
sed -i.bak 's|import "k8s.io/apiextensions-apiserver/|//import "k8s.io/apiextensions-apiserver/|' k8s.io/api/core/v1/generated.proto
rm k8s.io/api/core/v1/generated.proto.bak
make[1]: Leaving directory '/workspace/source/proto/k8s'
pip install -r dev_requirements.txt
Collecting pytest==5.3.1
  Downloading pytest-5.3.1-py3-none-any.whl (233 kB)
Requirement already satisfied: pytest-xdist==1.30.0 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 2)) (1.30.0)
Collecting pytest-cov==2.8.1
  Downloading pytest_cov-2.8.1-py2.py3-none-any.whl (18 kB)
Requirement already satisfied: flaky==3.6.1 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 4)) (3.6.1)
Collecting tenacity==6.0.0
  Downloading tenacity-6.0.0-py2.py3-none-any.whl (24 kB)
Requirement already satisfied: ipython==7.13.0 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 8)) (7.13.0)
Requirement already satisfied: nbconvert==5.6.1 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 9)) (5.6.1)
Requirement already satisfied: alibi==0.3.2 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 10)) (0.3.2)
Requirement already satisfied: matplotlib==3.1.3 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 11)) (3.1.3)
Requirement already satisfied: tensorflow==1.15.2 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 12)) (1.15.2)
Collecting coverage==4.5.4
  Downloading coverage-4.5.4-cp36-cp36m-manylinux1_x86_64.whl (205 kB)
Requirement already satisfied: attrs>=17.4.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (19.3.0)
Requirement already satisfied: packaging in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (20.3)
Requirement already satisfied: more-itertools>=4.0.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (8.2.0)
Requirement already satisfied: importlib-metadata>=0.12; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (1.6.0)
Requirement already satisfied: py>=1.5.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (1.8.1)
Requirement already satisfied: pluggy<1.0,>=0.12 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (0.13.1)
Requirement already satisfied: wcwidth in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (0.1.9)
Requirement already satisfied: six in /usr/local/lib/python3.6/site-packages (from pytest-xdist==1.30.0->-r dev_requirements.txt (line 2)) (1.14.0)
Requirement already satisfied: execnet>=1.1 in /usr/local/lib/python3.6/site-packages (from pytest-xdist==1.30.0->-r dev_requirements.txt (line 2)) (1.7.1)
Requirement already satisfied: pytest-forked in /usr/local/lib/python3.6/site-packages (from pytest-xdist==1.30.0->-r dev_requirements.txt (line 2)) (1.1.3)
Requirement already satisfied: pygments in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (2.6.1)
Requirement already satisfied: pexpect; sys_platform != "win32" in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (4.8.0)
Requirement already satisfied: decorator in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (4.4.2)
Requirement already satisfied: backcall in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.1.0)
Requirement already satisfied: traitlets>=4.2 in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (4.3.3)
Requirement already satisfied: prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (3.0.5)
Requirement already satisfied: setuptools>=18.5 in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (46.3.0)
Requirement already satisfied: jedi>=0.10 in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.17.0)
Requirement already satisfied: pickleshare in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.7.5)
Requirement already satisfied: bleach in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (3.1.5)
Requirement already satisfied: entrypoints>=0.2.2 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.3)
Requirement already satisfied: defusedxml in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.6.0)
Requirement already satisfied: nbformat>=4.4 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (5.0.6)
Requirement already satisfied: jupyter-core in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (4.6.3)
Requirement already satisfied: jinja2>=2.4 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (2.11.2)
Requirement already satisfied: pandocfilters>=1.4.1 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (1.4.2)
Requirement already satisfied: testpath in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.4.4)
Requirement already satisfied: mistune<2,>=0.8.1 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.8.4)
Requirement already satisfied: requests in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.23.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.0.1)
Requirement already satisfied: numpy in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.18.4)
Requirement already satisfied: Pillow in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (7.1.1)
Requirement already satisfied: scikit-learn in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.23.0)
Requirement already satisfied: spacy in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.2.4)
Requirement already satisfied: scikit-image in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.17.2)
Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (4.9.0)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /usr/local/lib/python3.6/site-packages (from matplotlib==3.1.3->-r dev_requirements.txt (line 11)) (2.4.7)
Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.6/site-packages (from matplotlib==3.1.3->-r dev_requirements.txt (line 11)) (1.2.0)
Requirement already satisfied: python-dateutil>=2.1 in /usr/local/lib/python3.6/site-packages (from matplotlib==3.1.3->-r dev_requirements.txt (line 11)) (2.8.1)
Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.6/site-packages (from matplotlib==3.1.3->-r dev_requirements.txt (line 11)) (0.10.0)
Requirement already satisfied: absl-py>=0.7.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.9.0)
Requirement already satisfied: astor>=0.6.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.8.1)
Requirement already satisfied: tensorflow-estimator==1.15.1 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.15.1)
Requirement already satisfied: wheel>=0.26; python_version >= "3" in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.34.2)
Requirement already satisfied: keras-preprocessing>=1.0.5 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.1.2)
Requirement already satisfied: opt-einsum>=2.3.2 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (3.2.1)
Requirement already satisfied: protobuf>=3.6.1 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (3.11.3)
Requirement already satisfied: tensorboard<1.16.0,>=1.15.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.15.0)
Requirement already satisfied: google-pasta>=0.1.6 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.2.0)
Requirement already satisfied: grpcio>=1.8.6 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.28.1)
Requirement already satisfied: gast==0.2.2 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.2.2)
Requirement already satisfied: wrapt>=1.11.1 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.12.1)
Requirement already satisfied: keras-applications>=1.0.8 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.0.8)
Requirement already satisfied: termcolor>=1.1.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.1.0)
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/site-packages (from importlib-metadata>=0.12; python_version < "3.8"->pytest==5.3.1->-r dev_requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: apipkg>=1.4 in /usr/local/lib/python3.6/site-packages (from execnet>=1.1->pytest-xdist==1.30.0->-r dev_requirements.txt (line 2)) (1.5)
Requirement already satisfied: ptyprocess>=0.5 in /usr/local/lib/python3.6/site-packages (from pexpect; sys_platform != "win32"->ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.6.0)
Requirement already satisfied: ipython-genutils in /usr/local/lib/python3.6/site-packages (from traitlets>=4.2->ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.2.0)
Requirement already satisfied: parso>=0.7.0 in /usr/local/lib/python3.6/site-packages (from jedi>=0.10->ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.7.0)
Requirement already satisfied: webencodings in /usr/local/lib/python3.6/site-packages (from bleach->nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.5.1)
Requirement already satisfied: jsonschema!=2.5.0,>=2.4 in /usr/local/lib/python3.6/site-packages (from nbformat>=4.4->nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (3.2.0)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/site-packages (from jinja2>=2.4->nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (1.1.1)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests->alibi==0.3.2->-r dev_requirements.txt (line 10)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.25.9)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.9)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2020.4.5.1)
Requirement already satisfied: pytz>=2017.2 in /usr/local/lib/python3.6/site-packages (from pandas->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2020.1)
Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.6/site-packages (from scikit-learn->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.0.0)
Requirement already satisfied: scipy>=0.19.1 in /usr/local/lib/python3.6/site-packages (from scikit-learn->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.4.1)
Requirement already satisfied: joblib>=0.11 in /usr/local/lib/python3.6/site-packages (from scikit-learn->alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.14.1)
Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.0.2)
Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (3.0.2)
Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.1.3)
Requirement already satisfied: srsly<1.1.0,>=1.0.2 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.0.2)
Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.0.0)
Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (7.4.0)
Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.4.1)
Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.6.0)
Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (4.46.0)
Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.0.3)
Requirement already satisfied: PyWavelets>=1.1.1 in /usr/local/lib/python3.6/site-packages (from scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.1.1)
Requirement already satisfied: imageio>=2.3.0 in /usr/local/lib/python3.6/site-packages (from scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.8.0)
Requirement already satisfied: tifffile>=2019.7.26 in /usr/local/lib/python3.6/site-packages (from scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2020.5.11)
Requirement already satisfied: networkx>=2.0 in /usr/local/lib/python3.6/site-packages (from scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.4)
Requirement already satisfied: soupsieve>1.2 in /usr/local/lib/python3.6/site-packages (from beautifulsoup4->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.0)
Requirement already satisfied: markdown>=2.6.8 in /usr/local/lib/python3.6/site-packages (from tensorboard<1.16.0,>=1.15.0->tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (3.2.2)
Requirement already satisfied: werkzeug>=0.11.15 in /usr/local/lib/python3.6/site-packages (from tensorboard<1.16.0,>=1.15.0->tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.0.1)
Requirement already satisfied: h5py in /usr/local/lib/python3.6/site-packages (from keras-applications>=1.0.8->tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (2.10.0)
Requirement already satisfied: pyrsistent>=0.14.0 in /usr/local/lib/python3.6/site-packages (from jsonschema!=2.5.0,>=2.4->nbformat>=4.4->nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.16.0)
Requirement already satisfied: imagecodecs>=2020.2.18 in /usr/local/lib/python3.6/site-packages (from tifffile>=2019.7.26->scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2020.2.18)
Installing collected packages: pytest, coverage, pytest-cov, tenacity
  Attempting uninstall: pytest
    Found existing installation: pytest 5.4.2
    Uninstalling pytest-5.4.2:
      Successfully uninstalled pytest-5.4.2
  Attempting uninstall: coverage
    Found existing installation: coverage 5.1
    Uninstalling coverage-5.1:
      Successfully uninstalled coverage-5.1
  Attempting uninstall: pytest-cov
    Found existing installation: pytest-cov 2.9.0
    Uninstalling pytest-cov-2.9.0:
      Successfully uninstalled pytest-cov-2.9.0
  Attempting uninstall: tenacity
    Found existing installation: tenacity 6.2.0
    Uninstalling tenacity-6.2.0:
      Successfully uninstalled tenacity-6.2.0
Successfully installed coverage-4.5.4 pytest-5.3.1 pytest-cov-2.8.1 tenacity-6.0.0
WARNING: You are using pip version 20.1; however, version 20.1.1 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
pip install -e ../../python
Obtaining file:///workspace/source/python
Requirement already satisfied: Flask<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (1.1.2)
Requirement already satisfied: Flask-cors<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (3.0.8)
Requirement already satisfied: redis<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (3.5.3)
Requirement already satisfied: requests<3.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (2.23.0)
Requirement already satisfied: numpy<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (1.18.4)
Requirement already satisfied: flatbuffers<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (1.12)
Requirement already satisfied: protobuf<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (3.11.3)
Requirement already satisfied: grpcio<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (1.28.1)
Requirement already satisfied: Flask-OpenTracing<1.2.0,>=1.1.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (1.1.0)
Requirement already satisfied: opentracing<2.3.0,>=2.2.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (2.2.0)
Requirement already satisfied: jaeger-client<4.2.0,>=4.1.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (4.1.0)
Requirement already satisfied: grpcio-opentracing<1.2.0,>=1.1.4 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (1.1.4)
Requirement already satisfied: pyaml<20.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (19.12.0)
Requirement already satisfied: gunicorn<20.1.0,>=19.9.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (20.0.4)
Requirement already satisfied: minio<6.0.0,>=4.0.9 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (5.0.10)
Requirement already satisfied: azure-storage-blob<3.0.0,>=2.0.1 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (2.1.0)
Requirement already satisfied: setuptools>=41.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (46.3.0)
Requirement already satisfied: prometheus_client<0.8.0,>=0.7.1 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.1.1rc0) (0.7.1)
Requirement already satisfied: itsdangerous>=0.24 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.1.1rc0) (1.1.0)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.1.1rc0) (2.11.2)
Requirement already satisfied: Werkzeug>=0.15 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.1.1rc0) (1.0.1)
Requirement already satisfied: click>=5.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.1.1rc0) (7.1.2)
Requirement already satisfied: Six in /usr/local/lib/python3.6/site-packages (from Flask-cors<4.0.0->seldon-core==1.1.1rc0) (1.14.0)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.1.1rc0) (1.25.9)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.1.1rc0) (2020.4.5.1)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.1.1rc0) (2.9)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.1.1rc0) (3.0.4)
Requirement already satisfied: threadloop<2,>=1 in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.1.1rc0) (1.0.2)
Requirement already satisfied: tornado<6,>=4.3 in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.1.1rc0) (5.1.1)
Requirement already satisfied: thrift in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.1.1rc0) (0.13.0)
Requirement already satisfied: PyYAML in /usr/local/lib/python3.6/site-packages (from pyaml<20.0.0->seldon-core==1.1.1rc0) (5.3.1)
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.1.1rc0) (2.8.1)
Requirement already satisfied: pytz in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.1.1rc0) (2020.1)
Requirement already satisfied: configparser in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.1.1rc0) (5.0.0)
Requirement already satisfied: azure-storage-common~=2.1 in /usr/local/lib/python3.6/site-packages (from azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.1.1rc0) (2.1.0)
Requirement already satisfied: azure-common>=1.1.5 in /usr/local/lib/python3.6/site-packages (from azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.1.1rc0) (1.1.25)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/site-packages (from Jinja2>=2.10.1->Flask<2.0.0->seldon-core==1.1.1rc0) (1.1.1)
Requirement already satisfied: cryptography in /usr/local/lib/python3.6/site-packages (from azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.1.1rc0) (2.9.2)
Requirement already satisfied: cffi!=1.11.3,>=1.8 in /usr/local/lib/python3.6/site-packages (from cryptography->azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.1.1rc0) (1.14.0)
Requirement already satisfied: pycparser in /usr/local/lib/python3.6/site-packages (from cffi!=1.11.3,>=1.8->cryptography->azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.1.1rc0) (2.20)
Installing collected packages: seldon-core
  Attempting uninstall: seldon-core
    Found existing installation: seldon-core 1.1.1rc0
    Uninstalling seldon-core-1.1.1rc0:
      Successfully uninstalled seldon-core-1.1.1rc0
  Running setup.py develop for seldon-core
Successfully installed seldon-core
WARNING: You are using pip version 20.1; however, version 20.1.1 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
# Run the core tests in parallel
pytest \
	--verbose \
	-s \
	-W ignore \
	-n "4" \
	-m "not sequential and not notebooks" 2>&1
============================= test session starts ==============================
platform linux -- Python 3.6.10, pytest-5.3.1, py-1.8.1, pluggy-0.13.1 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /workspace/source/testing/scripts, inifile: setup.cfg
plugins: cov-2.8.1, xdist-1.30.0, forked-1.1.3, flaky-3.6.1
gw0 I / gw1 I / gw2 I / gw3 I
[gw0] linux Python 3.6.10 cwd: /workspace/source/testing/scripts
[gw1] linux Python 3.6.10 cwd: /workspace/source/testing/scripts
[gw2] linux Python 3.6.10 cwd: /workspace/source/testing/scripts
[gw3] linux Python 3.6.10 cwd: /workspace/source/testing/scripts
[gw1] Python 3.6.10 (default, May 14 2020, 12:39:37)  -- [GCC 8.3.0]
[gw0] Python 3.6.10 (default, May 14 2020, 12:39:37)  -- [GCC 8.3.0]
[gw2] Python 3.6.10 (default, May 14 2020, 12:39:37)  -- [GCC 8.3.0]
[gw3] Python 3.6.10 (default, May 14 2020, 12:39:37)  -- [GCC 8.3.0]
gw0 [32] / gw1 [32] / gw2 [32] / gw3 [32]

scheduling tests via LoadScheduling

test_bad_graphs.py::TestBadGraphs::test_duplicate_predictor_name 
test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha3] 
test_api_version.py::test_api_version[machinelearning.seldon.io/v1] 
test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha2] Error from server (SeldonDeployment.machinelearing.seldon.io "dupname" is invalid: [spec.predictors[1]: Invalid value: "mymodel": Duplicate predictor name, spec: Invalid value: "mymodel": Traffic must sum to 100 for multiple predictors]): error when creating "../resources/bad_duplicate_predictor_name.json": admission webhook "v1alpha2.vseldondeployment.kb.io" denied the request: SeldonDeployment.machinelearing.seldon.io "dupname" is invalid: [spec.predictors[1]: Invalid value: "mymodel": Duplicate predictor name, spec: Invalid value: "mymodel": Traffic must sum to 100 for multiple predictors]

[gw3] PASSED test_bad_graphs.py::TestBadGraphs::test_duplicate_predictor_name 
test_helm_charts_clusterwide.py::TestClusterWide::test_abtest_model Error: uninstall: Release not loaded: mymodel: release: not found

[gw0] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha3] Error: uninstall: Release not loaded: mymodel: release: not found

[gw2] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1] Error: uninstall: Release not loaded: mymodel: release: not found

[gw1] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha2] Error: uninstall: Release not loaded: myabtest: release: not found

[gw3] PASSED test_helm_charts_clusterwide.py::TestClusterWide::test_abtest_model 
test_helm_charts_clusterwide.py::TestClusterWide::test_single_model 
test_bad_graphs.py::TestBadGraphs::test_model_name_mismatch 
test_bad_graphs.py::TestBadGraphs::test_model_no_graph Error from server (SeldonDeployment.machinelearing.seldon.io "namemismatch" is invalid: spec.predictors[0].graph: Invalid value: "complex-model_bad_name": Can't find container for Predictive Unit): error when creating "../resources/bad_name_mismatch.json": admission webhook "v1alpha2.vseldondeployment.kb.io" denied the request: SeldonDeployment.machinelearing.seldon.io "namemismatch" is invalid: spec.predictors[0].graph: Invalid value: "complex-model_bad_name": Can't find container for Predictive Unit
Error from server (SeldonDeployment.machinelearing.seldon.io "test-deployment" is invalid: spec.predictors[0].graph: Invalid value: "": Can't find container for Predictive Unit): error when creating "../resources/bad_no_graph.yaml": admission webhook "v1.vseldondeployment.kb.io" denied the request: SeldonDeployment.machinelearing.seldon.io "test-deployment" is invalid: spec.predictors[0].graph: Invalid value: "": Can't find container for Predictive Unit

[gw1] PASSED test_bad_graphs.py::TestBadGraphs::test_model_name_mismatch 
[gw0] PASSED test_bad_graphs.py::TestBadGraphs::test_model_no_graph 
test_prepackaged_servers.py::TestPrepack::test_sklearn 
test_prepackaged_servers.py::TestPrepack::test_tfserving Error: uninstall: Release not loaded: mymodel: release: not found

[gw2] PASSED test_helm_charts_clusterwide.py::TestClusterWide::test_single_model 
test_local_operators.py::TestLocalOperators::test_labelled_operator Error from server (InternalError): error when creating "../resources/model_controller_id.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.test-labelled-operator.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.110.86.22:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/model_controller_id.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.test-labelled-operator.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.110.86.22:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/model_controller_id.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.test-labelled-operator.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.110.86.22:443: connect: connection refused

test_helm_charts_clusterwide.py::TestClusterWide::test_mab_model Error: uninstall: Release not loaded: mymab: release: not found

[gw3] PASSED test_helm_charts_clusterwide.py::TestClusterWide::test_mab_model 
[gw0] FAILED test_prepackaged_servers.py::TestPrepack::test_tfserving 
[gw1] PASSED test_prepackaged_servers.py::TestPrepack::test_sklearn 
[gw2] PASSED test_local_operators.py::TestLocalOperators::test_labelled_operator 
test_prepackaged_servers.py::TestPrepack::test_text_alibi_explainer 
test_prepackaged_servers.py::TestPrepack::test_xgboost 
test_local_operators.py::TestLocalOperators::test_namespace_operator 
test_prepackaged_servers.py::TestPrepack::test_mlflow 
[gw3] PASSED test_local_operators.py::TestLocalOperators::test_namespace_operator 
[gw2] FAILED test_prepackaged_servers.py::TestPrepack::test_text_alibi_explainer 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-istio] 
test_prepackaged_servers.py::TestPrepack::test_openapi_sklearn 
[gw1] PASSED test_prepackaged_servers.py::TestPrepack::test_xgboost 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-ambas] 
[gw2] PASSED test_prepackaged_servers.py::TestPrepack::test_openapi_sklearn 
[gw0] PASSED test_prepackaged_servers.py::TestPrepack::test_mlflow 
[gw1] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph4.json-True-ambas] 
[gw3] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-istio] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph3.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph4.json-True-istio] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph3.json-True-istio] 
[gw2] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph4.json-True-ambas] 
[gw0] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph3.json-True-ambas] 
[gw1] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph4.json-True-istio] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph8.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph5.json-True-istio] 
[gw3] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph3.json-True-istio] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph5.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph8.json-True-istio] 
[gw1] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph8.json-True-ambas] 
[gw2] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph5.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph7.json-graph8.json-False-ambas] 
[gw0] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph5.json-True-istio] 
[gw3] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph8.json-True-istio] 
test_security.py::test_xss_escaping 
test_rolling_updates.py::test_rolling_deployment[graph7.json-graph8.json-False-istio] 
[gw1] PASSED test_rolling_updates.py::test_rolling_deployment[graph7.json-graph8.json-False-ambas] 
test_security.py::test_xss_header 
[gw2] PASSED test_security.py::test_xss_escaping 
[gw0] PASSED test_rolling_updates.py::test_rolling_deployment[graph7.json-graph8.json-False-istio] 
test_tracing.py::test_tracing_rest 
[gw3] PASSED test_security.py::test_xss_header 
[gw1] PASSED test_tracing.py::test_tracing_rest 

=================================== FAILURES ===================================
__________________________ TestPrepack.test_tfserving __________________________
[gw0] linux -- Python 3.6.10 /usr/local/bin/python

self = <test_prepackaged_servers.TestPrepack object at 0x7ff4c3e54d68>
namespace = 'test-tfserving'

    def test_tfserving(self, namespace):
        spec = "../../servers/tfserving/samples/mnist_rest.yaml"
        retry_run(f"kubectl apply -f {spec}  -n {namespace}")
        wait_for_status("tfserving", namespace)
        wait_for_rollout("tfserving", namespace)
        time.sleep(1)
        logging.warning("Initial request")
        r = initial_rest_request(
            "tfserving",
            namespace,
            data=[create_random_data(784)[1].tolist()],
            dtype="ndarray",
        )
>       assert r.status_code == 200
E       AttributeError: 'NoneType' object has no attribute 'status_code'

test_prepackaged_servers.py:56: AttributeError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:test_prepackaged_servers.py:49 Initial request
WARNING  root:seldon_e2e_utils.py:209 Bad status:404
WARNING  root:seldon_e2e_utils.py:209 Bad status:404
WARNING  root:seldon_e2e_utils.py:209 Bad status:404
WARNING  root:seldon_e2e_utils.py:209 Bad status:404
____________________ TestPrepack.test_text_alibi_explainer _____________________
[gw2] linux -- Python 3.6.10 /usr/local/bin/python

self = <test_prepackaged_servers.TestPrepack object at 0x7fe82e44acf8>
namespace = 'test-text-alibi-explainer'

    def test_text_alibi_explainer(self, namespace):
        spec = "../resources/movies-text-explainer.yaml"
        retry_run(f"kubectl apply -f {spec} -n {namespace}")
        wait_for_status("movie", namespace)
        wait_for_rollout("movie", namespace, expected_deployments=2)
        time.sleep(5)
        logging.warning("Initial request")
        r = initial_rest_request(
            "movie", namespace, data=["This is test data"], dtype="ndarray"
        )
        assert r.status_code == 200
        e = rest_request(
            "movie",
            namespace,
            data=["This is test data"],
            dtype="ndarray",
            method="explain",
            predictor_name="movies-predictor",
        )
>       assert e.status_code == 200
E       AttributeError: 'NoneType' object has no attribute 'status_code'

test_prepackaged_servers.py:143: AttributeError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:165 Failed to find status - sleeping
WARNING  root:test_prepackaged_servers.py:130 Initial request
WARNING  root:seldon_e2e_utils.py:209 Bad status:404
===Flaky Test Report===

test_rolling_deployment[graph1.json-graph3.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph5.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph7.json-graph8.json-False-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph4.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph5.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph2.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph4.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph8.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph7.json-graph8.json-False-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph2.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph3.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph8.json-True-istio] passed 1 out of the required 1 times. Success!

===End Flaky Test Report===
================== 2 failed, 30 passed in 1921.17s (0:32:01) ===================
make: *** [Makefile:125: test_parallel] Error 1
Test returned errors
kind delete cluster
Deleting cluster "kind" ...
Stopping Docker: dockerProgram process in pidfile '/var/run/docker-ssd.pid', 1 process(es), refused to die.
[31m
Pipeline failed on stage 'end-to-end' : container 'step-test-end-to-end'. The execution of the pipeline has stopped.[0m
